Build.cs

```diff
@@ -511,7 +511,7 @@ private string GetThemeVersion(string packageJsonPath)
                     Console.Write($"Are you sure you want to release {GitRepository.Identifier}? (y/N): ");
                     var response = Console.ReadLine();
 
-                    if (string.Compare(response, "y", true, CultureInfo.InvariantCulture) != 0)
+                    if (string.Equal(response, "y", StringComparison.OrdinalIgnoreCase))
```

Comment:
- Are you sure you want to release {GitRepository.Identifier}? (y/N):
- y
- Aborted